spec/requests/api/v1/users/create_spec.rb

```diff
@@ -1,97 +1,93 @@
-describe 'POST api/v1/users/', type: :request do
-  let(:user)            { User.last }
-  let(:failed_response) { 422 }
-
-  describe 'POST create' do
-    subject { post user_registration_path, params:, as: :json }
-
-    let(:username)              { 'test' }
-    let(:email)                 { 'test@test.com' }
-    let(:password)              { '12345678' }
-    let(:password_confirmation) { '12345678' }
-    let(:first_name)            { 'Johnny' }
-    let(:last_name)             { 'Perez' }
-
-    let(:params) do
-      {
-        user: {
-          username:,
-          email:,
-          password:,
-          password_confirmation:,
-          first_name:,
-          last_name:
-        }
+# frozen_string_literal: true
+
+describe 'POST api/v1/users' do
+  subject { post user_registration_path, params:, as: :json }
+
+  let(:user) { User.last }
+  let(:email) { 'test@test.com' }
+
+  let(:params) do
+    {
+      user: {
+        username: 'test',
+        email:,
+        password: '12345678',
+        password_confirmation: '12345678',
+        first_name: 'Johnny',
+        last_name: 'Perez'
       }
-    end
+    }
+  end
 
-    it_behaves_like 'does not check authenticity token'
-    it_behaves_like 'there must not be a Set-Cookie in Header'
+  it_behaves_like 'does not check authenticity token'
+  it_behaves_like 'there must not be a Set-Cookie in Header'
 
-    it 'returns a successful response' do
-      subject
-      expect(response).to have_http_status(:success)
-    end
+  it 'returns a successful response' do
+    subject
+    expect(response).to have_http_status(:success)
+  end
 
-    it 'creates the user' do
-      expect { subject }.to change(User, :count).by(1)
+  it 'creates the user' do
+    expect { subject }.to change(User, :count).by(1)
+  end
+
+  it 'returns the user' do
```

Comment:
https://rspec.rubystyle.guide/#expectation-per-example
```suggestion
  it 'returns the user data', :aggregate_failures do
```